core/sql/regress/seabase/EXPECTED040

```diff
@@ -227,6 +262,122 @@ CREATE TABLE TRAFODION.T040SCH.T040NOSALTSAME
+
+--- SQL operation complete.
+>>
+>>showddl t040limitedcols;
+
+CREATE TABLE TRAFODION.T040SCH.T040LIMITEDCOLS
+  (
+    A                                INT NO DEFAULT NOT NULL NOT DROPPABLE
+  , B                                CHAR(256) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL
+  , C                                VARCHAR(256) CHARACTER SET ISO88591
+      COLLATE DEFAULT DEFAULT NULL
+  , D                                CHAR(256 BYTES) CHARACTER SET UTF8 COLLATE
+      DEFAULT DEFAULT NULL
+  , E                                VARCHAR(256 BYTES) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL
+  , F                                CHAR(128) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , G                                VARCHAR(128) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , PRIMARY KEY (A ASC)
+  )
+ ATTRIBUTES ALIGNED FORMAT
+;
+
+--- SQL operation complete.
+>>
+>>create table t040somelimitedcols like t040bigcols 
++>limit column length to 1002;
+
+--- SQL operation complete.
+>>
+>>showddl t040somelimitedcols;
+
+CREATE TABLE TRAFODION.T040SCH.T040SOMELIMITEDCOLS
+  (
+    A                                INT NO DEFAULT NOT NULL NOT DROPPABLE
+  , B                                CHAR(1000) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL
+  , C                                VARCHAR(1001) CHARACTER SET ISO88591
+      COLLATE DEFAULT DEFAULT NULL
+  , D                                CHAR(1002 BYTES) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL
+  , E                                VARCHAR(1002 BYTES) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL
+  , F                                CHAR(501) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , G                                VARCHAR(501) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , PRIMARY KEY (A ASC)
+  )
+ ATTRIBUTES ALIGNED FORMAT
+;
+
+--- SQL operation complete.
+>>
+>>create table t040fewerlimitedcols like t040bigcols 
++>limit column length to 2010;
+
+--- SQL operation complete.
+>>
+>>showddl t040fewerlimitedcols;
+
+CREATE TABLE TRAFODION.T040SCH.T040FEWERLIMITEDCOLS
+  (
+    A                                INT NO DEFAULT NOT NULL NOT DROPPABLE
+  , B                                CHAR(1000) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL
+  , C                                VARCHAR(1001) CHARACTER SET ISO88591
+      COLLATE DEFAULT DEFAULT NULL
+  , D                                CHAR(2010 BYTES) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL
+  , E                                VARCHAR(2010 BYTES) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL
+  , F                                CHAR(1004) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , G                                VARCHAR(1005) CHARACTER SET UCS2 COLLATE
+      DEFAULT DEFAULT NULL
+  , PRIMARY KEY (A ASC)
+  )
+ ATTRIBUTES ALIGNED FORMAT
```

Comment:
Good suggestion. As it happens, test seabase/TEST011 has an example of a salted table with a long column which exercises this case. (I found it when doing initial regression runs. It cored due to a bug in an earlier draft of these changes.)